### ISSUE TITLE
[NFR] Envio de SMS após ligação

### ISSUE THREAD

**Issue by jhonata-menezes** [haskellcamargo/gemidao-do-zap#22]
Por experiência própria e bom enviar um SMS com o número real de quem fez a ligação, para evitar possíveis brigas, tenho um serviço com o mesmo propósito: https://chamadagemidao.com

**Comment by iagocaldeira** [comment]
Acho que no caso dessa API vai muito também de esperar que o usuário tenha boa fé na hora de utilizar o sistema, mas acho uma boa opção oferecer essa possibilidade

**Comment by jhonata-menezes** [comment]
Realmente tem que ter.
Hoje teve uma pessoa que entrou no suporte e perguntou se tinha como saber quem realizou a ligação, pois ligaram a partir do número da namorada dele e ele ficou preocupado, mas não tinha sido a partir do meu serviço.

**Comment by StanleySathler** [comment]
É complicado. Apesar do propósito do serviço ser realmente a zoeira, @jhonata-menezes, o caso do sujeito que ficou preocupado com o número da namorada é passível de acontecer. Se a pessoa não conhece o gemidão, pode até achar que aconteceu algo bem ruim com o responsável pelo número utilizado.

O problema neste caso é que esta aplicação é, diferente de um serviço, de código aberto e de fácil modificação. Ou seja, se o usuário ainda ter má fé, ele consegue remover o envio desse SMS. Ou seja, não poderíamos considerar o envio como um mecanismo de "segurança".

A minha sugestão então, aprimorando um pouco a issue, é incluir por padrão o envio do SMS e um parâmetro para desabilitá-lo. Assim, deixamos a opção para aqueles que querem a "zoeira do bem" sem forçá-lo a precisar modificar o código.

**Comment by jhonata-menezes** [comment]
Concordo Stanley, quem quiser pode modificar.

**Comment by lucasdamo** [comment]
Como que funciona esse tipo de serviço e como que conseguem ligar fingindo ser qualquer numero ?

**Comment by jhonata-menezes** [comment]
É por este serviço as ligações: http://www.totalvoice.com.br

**Comment by lucasdamo** [comment]
Sim, mas como o TotalVoice funciona ? Não era para existir um meio de proteção contra isso, regulado pela anatel sei la ?

**Comment by santiagomarcos** [comment]
@lucasdamo ele funciona como serviço de voip, conta sip. é uma linha digital só pesquisar. varias empresas de PABX usa.
